### ISSUE TITLE
Establish whether weight decay removes the horizon ceiling

### ISSUE THREAD

**Issue by mmmfrieddough** [mmmfrieddough/anthro-chess#562]
## Goal

Establish whether bounding parameter growth removes the horizon ceiling
`docs/decisions/0088-the-horizon-has-a-ceiling-and-it-is-counted-in-steps.md`
measured, at a width and step count where that ceiling actually appears.

`#490` found that a width-128 run stops improving near 222,000 optimizer steps
and is clearly degraded by 347,000, on both loss and accuracy and at two learning
rates. The evidence points at unbounded parameter growth under
`weight_decay = 0.0`, but no arm has been run with decay enabled where the
degradation exists, so the cause is evidenced rather than demonstrated.

## Why This Blocks The Ladder

At 800 positions per parameter the ceiling is reached near 4.55e6 parameters,
about `model_dim` 240. `#54` fits a ladder spanning one to two decades of size at
a fixed ratio, which puts its upper rungs past the ceiling and its lower rungs
far below it. That biases the fitted size exponent, and
`docs/research.md` (Resolving Discrepancies In Compute-Optimal Scaling) records
that no goodness-of-fit statistic detects this class of error.

The target is affected directly. At the same 800 positions per parameter the
vehicle runs 69,466 steps and the target runs 1,007,941, which is nearly three
times past where width 128 was visibly degraded.

## What Is Already Known

- Parameter norm grows 13.05x over 520,800 steps with no sign of saturation, and
  the output head grows most at 16.4x. Its scale sets how sharp the predicted
  distribution is.
- The symptom is miscalibration rather than lost capability: loss degrades 1.94
  percent while top-1 accuracy is flat, and `legality.mask_penalty` is unmoved.
- No instability. `clip_rate` is 0 at every cadence and gradient norms rise only
  from 0.21 to 0.49.
- Growth is per-step and width independent: 2.23x at width 32 against 2.20x at
  width 128 at matched steps.
- A width-32 arm at a 1x-horizon timescale bounded growth as predicted, 3.32x
  against 2.67x, and moved loss by 0.06 percent. That arm reached only 56,880
  steps, far short of the failure, so it neither confirms nor refutes anything.

## The Rule Is Stated Over The Wrong Invariant

`anthro_chess.training.scaling_rules` sets the decay timescale as a multiple of
the horizon, so the per-step decay rate is `1 / (horizons * steps)` and weakens
as a run lengthens. Equilibrium norm depends on the per-step rate, so a longer
run is given less of exactly what bounds it. Published practice holds a constant
coefficient instead, which keeps that rate invariant to the horizon. Whether to
change the rule is part of this issue rather than assumed by it.

## Acceptance Criteria

- One arm at `model_dim` 128 with decay enabled, trained to at least 350,000
  optimizer steps, compared against `h490-trunk-0p003` which already carries
  uncooled pool readings at seven step counts under the same configuration.
- Parameter norm is reported at matched steps for both arms, so that the arm is
  shown to have bounded growth rather than assumed to have.
- The reading states whether the loss turnaround is removed, moved, or unchanged.
  A null is a result and closes this issue.
- Where decay removes the ceiling, the issue says what the rule should be and
  whether the timescale form survives, per the section above.
- Whether the vehicle adopts the change is decided explicitly and separately.
  `weight_decay` is inside `training_sha256`, so changing it invalidates the
  stored seed dispersion and every candidate arm read against it, which
  `0065` and `0076` price at five arms plus the loss of prior comparisons.

## Carrying Metric

**Shape: several named metrics.** The claim is that a defect is removed, so it is
a conjunction: growth must be bounded and the turnaround must not appear.

| metric | benchmark | floor today |
| --- | --- | --- |
| `held_out.move_loss` | `anthro eval run` | prints one |
| `held_out.top1_accuracy` | `anthro eval run` | prints one |

Parameter norm is read from the checkpoints directly and has no benchmark.

The comparison is against `h490-trunk-0p003` at matched steps rather than against
the vehicle, because the vehicle's horizon never reaches the regime in question.
Both arms carry different `training_sha256`, correctly, so no seed floor applies
and the pull request says so.

## Required Environment

One CUDA host with the widened corpus prepared. Roughly 30 GPU-hours for a
350,000-step arm at the measured 53,000 positions per second.

## References

- `docs/decisions/0088-the-horizon-has-a-ceiling-and-it-is-counted-in-steps.md`
- `#490` and its pull request, for the full curve and the mechanism evidence
- `#54`, which this blocks

**Comment by mmmfrieddough** [comment]
Claimed for implementation by Claude Code (Opus 5).

Session: 34f68598-084e-4363-9388-f23825bb7051

**Comment by mmmfrieddough** [comment]
Two arms are in flight rather than one, on the two cards of this host, for about 32 hours from now.

Both copy the control `h490-trunk-0p003` exactly and differ from it only in `weight_decay`, so all three share a data order, a seed, a declared horizon of 2,222,080 steps and a constant 3e-3 trunk rate:

```console
uv run anthro train --config configs/training/ablation-vehicle.toml \
  --set 'run_name="h562-wd-w128-0p1"' --set steps=2222080 \
  --set checkpoint_every_steps=6944 --set weight_decay=0.1

uv run anthro train --config configs/training/ablation-vehicle.toml \
  --set 'run_name="h562-wd-w128-0p01"' --set steps=2222080 \
  --set checkpoint_every_steps=6944 --set weight_decay=0.01
```

The acceptance criteria ask for one arm. A second is run because a single coefficient answers only whether decay removes the ceiling at that one strength, and the criterion that the reading say what the rule should be needs more than one point. Decoupled decay makes the meaningful quantity the per-step rate `learning_rate * weight_decay`, so the two arms are decay timescales of 3,333 and 33,333 optimizer steps. Both sit far below the 222,208-step loss minimum the control reaches, so both should bound growth well inside the run; the pair brackets one decade of strength. The gentler arm carries the same per-step rate that published runs quoting `weight_decay = 0.1` actually apply, since their peak rates are an order of magnitude below this one.

Neither arm is compared on wall-clock timing, so sharing the host costs the reading nothing. Measured throughput is 50,200 positions per second per card with both running.

Target is at least 354,144 steps, the 51st checkpoint, which clears the 350,000 the criteria ask for and covers the control's matched readings at 111,104, 166,656, 222,208, 277,760 and 347,200.
